script/Ciel/Day 12/12_01_CIEL12_3B.txt

```diff
@@ -306,12 +306,12 @@ I steal a glance at Ciel-senpai.
 [sha:b11ede70be08e4c2b71c4af33393b7346ad8911f]{
 -- Page 115, Offset 31163.
 -- この人は街で派手に遊ぶより、公園をのんびり散策したりする方が性に合っているような気がして、どうも誘えない。
-It's hard to ask her out since this person seems more suited to taking leisurely strolls at the park than playing around extravagantly at the city.
+I can't ask her out. After all, she seems more suited to taking a leisurely stroll at the park than lavishly playing around in the city.
```

Comment:
she seems more suited to a leisurely stroll in the park than she is playing around in the city